### PR TITLE
Remove information from manifest

### PR DESCRIPTION
**Pull request by jbech-linaro**
Since we gather all instructions at optee_os git we have removed the
information from this particular git.

Change-Id: I707f664a1f81781f66cc0a1cae50b74e66864ef5
Signed-off-by: Joakim Bech <joakim.bech@linaro.org>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,115 +1,7 @@
...
-(Ubuntu, Mint etc) you simply install it by running:
-```
-$ sudo apt-get install ccache
-```
-
-The helper makefiles are configured to automatically find and use ccache if
-ccache is installed on your system, so other than having it installed you don't
-have to think about anything.
+In the OP-TEE project we try to gather all documentation under the
+[optee_os](https://github.com/OP-TEE) git, therefore we recommend that you
```
Comment (pascal-brand-st-dev) [inline README.md]: the link points to OP-TEE, not optee_os

**Comment by pascal-brand-st-dev** [comment]
With or without my comment:

    Reviewed-by: Pascal Brand <pascal.brand@linaro.org>

File: README.md (hunk 1 above)
Comment (jbech-linaro) [inline README.md]: I'll update.

**Comment by jbech-linaro** [comment]
Patches squashed, tag applied, ready to be merged.

**Pull request merged**